tasks/eth/006-MCP-L1/README.md

```diff
@@ -0,0 +1,176 @@
+# Mainnet MCP L1 Upgrade
+
+Status: DRAFT, NOT READY TO SIGN
+
+> [!IMPORTANT] !!! DO NOT SIGN using this playbook yet, as the associated governance proposal has
+> not yet passed.
+
+## Objective
+
+This is the playbook for executing Mainnet MCP L1 Upgrade, pending approval by governance. Voting on this
+proposal is tracked at https://vote.optimism.io/proposals/tbd.
```

Comment:
Let's change this to match the format used in [upgrade 4](https://github.com/ethereum-optimism/superchain-ops/blob/5388e3e1bba3c916a077f286c08a915814e6a57d/tasks/eth/004-add-superchainConfig/README.md#L7):

```suggestion
```markdown
This is the playbook for executing Upgrade #6, Multi-Chain Prep (MCP) L1.

The proposal was:
- [X] Approved by Token House voting here: https://vote.optimism.io/proposals/47253113366919812831791422571513347073374828501432502648295761953879525315523
- [ ] Not vetoed by the Citizens' house here: https://snapshot.org/#/citizenshouse.eth/proposal/0x2bc6565053b73813c6b0a001c8c07eb5656234b4d8bae12ba6541250993d1d25
```
```